src/stores/modules/User.store.js

```diff
@@ -7,6 +7,7 @@ const state = {
   userData: {
     'username': localStorage.getItem('username'),
     'email': localStorage.getItem('email'),
+    'globalPermission': localStorage.getItem('globalPermission'),
```

Comment:
So there's only one value for "no global permission", if there's no 'globalPermission' entry here the value should get set to "off".